ERCS/erc-7204.md

```diff
@@ -0,0 +1,210 @@
+---
+eip: 7204
+title: Contract wallet management token
+description: Designed for simple token
+author: Xiang (@wenzhenxiang), Ben77 (@ben2077), Mingshi S. (@newnewsms)
+discussions-to: https://ethereum-magicians.org/t/token-asset-management-interface-with-smart-contract-wallet/14759
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-06-21
+requires: 165
+---
+
+## Abstract
+
+A proposal to manage simple tokens by the user's smart contract wallet, which provides a new way to manage assets, utilizes the programmability of the smart contract wallet, and also provides more playability.
+
+## Motivation
+
+EOA wallet has no state and code storage, and the smart contract wallet is different.
+
+AA is a direction of the smart contract wallet, which works around abstract accounts. This ERC can also be an extension based on [ERC-4337](./eip-4337.md) or as a plug-in for wallets.
+
+The smart contract wallet allows the user's own account to have state and code, bringing programmability to the wallet. We think there are more directions to expand. For example, token asset management, functional expansion of token transactions, etc.
+
+The smart contract wallet interface of this ERC is for asset management and asset approval. It supports the simpletoken ERC-X, and [ERC-20](./eip-20.md) is backward compatible with ERC-X, so it can be compatible with the management of all fungible tokens in the existing market.
```

Comment:
If you include HTML-style comments, the linter will make sure you replace them before going into Review.

```suggestion
The smart contract wallet interface of this ERC is for asset management and asset approval. It supports the simpletoken <!-- TODO --> ERC-X, and [ERC-20](./eip-20.md) is backward compatible with <!-- TODO --> ERC-X, so it can be compatible with the management of all fungible tokens in the existing market.
```